### PR TITLE
Adding swagger of sensitivity labels APIs.

### PR DESCRIPTION
**Pull request by bashahee**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by hovsepm** [comment]
@ravbhatnagar could you take a look to the new swagger spec?

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource": {
```
Comment (jaredmoo) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json]: Please manually remove these and reference common definitions. See other swagger files for examples.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+      }
+    }
+  },
+  "definitions": {
+    "SensitivityLabelProperties": {
+      "description": "Properties of a sensitivity label.",
+      "type": "object",
+      "properties": {
+        "labelName": {
+          "description": "The label name.",
```
Comment (jaredmoo) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json]: For labelName and informationType, is it freeform text or is there a list of allowed values? If latter - please add enum info.

Comment (bashahee) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json]: Former.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json (hunk 1 above)
Comment (bashahee) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json]: Done.

**Review by jaredmoo — APPROVED** [review]

**Comment by bashahee** [comment]
@hovsepm Can this PR be merged as it was approved?

**Comment by hovsepm** [comment]
@bashahee I'm waiting on ARM review.

**Review by hovsepm — APPROVED** [review]

**Comment by bashahee** [comment]
@hovsepm Can you please merge this PR since you approved it?

**Pull request merged**